utils/swift_build_support/swift_build_support/targets.py

```diff
@@ -284,5 +284,7 @@ def toolchain_path(install_destdir, install_prefix):
     built_toolchain_path = install_destdir
     if platform.system() == 'Darwin':
         # The prefix is an absolute path, so concatenate without os.path.
-        built_toolchain_path += darwin_toolchain_prefix(install_prefix)
+        built_toolchain_path += darwin_toolchain_prefix(install_prefix) + "/usr"
```

Comment:
I'm not sure if calling this a "toolchain path" is good terminology, since on Darwin that would mean the `.xctoolchain` directory.  That's also what the documentation string for this method says.  After this change, it's always an absolute form of the install_prefix.  I'm not sure what to call that 🤔

By the way, `darwin_toolchain_prefix` removes `usr` from `install_prefix`, and then this adds it back.

Reply:
>I'm not sure if calling this a "toolchain path" is good terminology

I have no idea why this is needed on Darwin or what the common paths are there, I just tried to keep the behavior the same.

>By the way, darwin_toolchain_prefix removes usr from install_prefix, and then this adds it back.

Yes, as I noted in my previous comment, that is what was being done everywhere already, so rather than adding it back in a dozen different places, I simply added it back once. :smile:

I figured it might be because `install_prefix` on Darwin doesn't necessarily end with `/usr`, so `darwin_toolchain_prefix()` just removes that last component and the rest of the code replaces it with `/usr`. But that's just a guess, given my professed ignorance of the Darwin toolchain.

Just let me know what should be done, as this pull actually starts using `install_prefix` on non-Darwin platforms, rather than assuming it's always `/usr`.